src/selectors/index.js

```diff
@@ -1,7 +1,11 @@
 import getAllProjectKeys from './getAllProjectKeys';
 import getAllProjects from './getAllProjects';
 import getCompiledProjects from './getCompiledProjects';
-import getConsoleHistory from './getConsoleHistory';
+import {
+  getConsoleHistory,
+  getPreviousConsoleInput,
+  getPreviousHistoryIndex,
+} from './getConsoleHistory';
```

Comment:
This breaks the pattern of `export default` in existing selectors but it seemed overly split up to have each of these functions in their own file. In my own limited experience I've been defining these inline